source/views/stoprint/print-release.tsx

```diff
@@ -81,7 +81,7 @@ export const PrintJobReleaseView = (): JSX.Element => {
 	let {job, printer} = route.params
 
 	let {data: usernameData, isLoading: loadingUsername} = useUsername()
-	let username = usernameData ? usernameData.username : ''
+	const username = usernameData ?? ''
```

Comment:
same here for inlining the default value and renaming from usernameData to username